build.gradle

```diff
@@ -1249,9 +1249,14 @@ if (gradle.gradleVersion != jfxGradleVersion) {
 
 // Look for stub runtime in bundled sdk, standalone sdk, or boot JDK
 
+// Allows automatic provisioning of webkit+media shared libraries from an official OpenJfx build
```

Comment:
If you are going to capitalize it, then it should be "OpenJFX".

Can you add "from Maven Central" ?